handshake/connection_parameters_manager.go

```diff
@@ -61,14 +63,19 @@ var (
 )
 
 // NewConnectionParamatersManager creates a new connection parameters manager
-func NewConnectionParamatersManager(pers protocol.Perspective, v protocol.VersionNumber) ConnectionParametersManager {
+func NewConnectionParamatersManager(
+	pers protocol.Perspective, v protocol.VersionNumber,
+	maxReceiveStreamFlowControlWindow protocol.ByteCount, maxReceiveConnectionFlowControlWindow protocol.ByteCount,
```

Comment:
Go 1.9 should fix that. On the other hand, I don't think the split between the quic package and the handshake package is justified right now anyway, but that's not an issue we need to resolve right now.